src/config/vnc_openstack/vnc_openstack/contrail_res_handler.py

```diff
@@ -80,7 +80,11 @@ class ResourceDeleteHandler(ContrailResourceHandler):
 
     def _resource_delete(self, id=None, fq_name=None):
         delete_method = getattr(self._vnc_lib, self.resource_delete_method)
-        delete_method(id=id, fq_name=fq_name)
+        try:
+            delete_method(id=id, fq_name=fq_name)
+        except vnc_exc.NoIdError:
+            raise db_handler.DBInterfaceV2._raise_contrail_exception(
```

Comment:
Babu I think its better we don't throw exception here. I think it would be better for the caller to catch and throw the exception. There could be cases when the caller wants to ignore NoIdError exception and continue with the rest of the code. What do you think ?